.github/workflows/ci.yml

```diff
@@ -202,7 +202,7 @@ jobs:
           uses: actions/checkout@v4
           with:
             repository: maticnetwork/matic-cli
-            ref: master
+            ref: v0.11.0
```

Comment:
Should we leave it as `master` so that it is able to reference the latest from matic-cli repo always?

Reply:
Hey @ssandeep 
With `master` currently it'd not work, because `matic-cli` enables `gRPC` by default.
There are two possible ways to solve this:
1. We wait for `gRPC` changes in `bor` and `heimdall` to be merged to `master` (next releases), then this will auto-solve, and we can revert back to `master`
2. We change the default behaviour of `matic-cli` to set `gRPC` to `false`. I set it to `true` because imho it was a good chance to test both `gRPC` (`bor`-`heimdall` comms) and `rpc` together

Reply:
Ok, thanks for the context @marcello33